system/Helpers/array_helper.php

```diff
@@ -45,14 +45,12 @@ function dot_array_search(string $index, array $array)
 	 * @param array $indexes
 	 * @param array $array
 	 *
-	 * @return mixed|null
+	 * @return mixed
 	 */
 	function _array_search_dot(array $indexes, array $array)
```

Comment:
If this has always been a "private" function can we go ahead and mark it `@internal`?

Reply:
Now marked `@internal`